Jenkinsfile

```diff
@@ -127,8 +127,8 @@ pipeline {
             }
             post {
                 always {
-                    warnings consoleParsers: [[parserName: 'CppLint']], canRunOnFailed: true
-                    warnings consoleParsers: [[parserName: 'math-dependencies']], canRunOnFailed: true
+                    recordIssues enabledForFailure: true, tool: cppLint()
+		    warnings consoleParsers: [[parserName: 'math-dependencies']], canRunOnFailed: true
```

Comment:
I had tried before. Just to make sure, I tried again. This is the doc that makes me think that we can't do it that way: https://github.com/jenkinsci/warnings-ng-plugin/blob/master/doc/Documentation.md#creating-a-new-tool-in-the-user-interface

It says that you can have a Groovy parser, it just needs to be configured using the web interface. I also tried those instructions, but to no avail. (I'm sure it can't find the groovy script in the right place.)

Reply:
Ah gotcha. Let me take a quick crack at it.

Reply:
Sorry, I wasn't clear. I followed the instructions to run the Groovy script, but didn't add it via the web interface. Hopefully it does work!

Reply:
I added it via the web interface and pushed the change to the Jenkinsfile. Will it show up as working in this build or do we have to trigger some tests for it?